flocker/acceptance/endtoend/test_diagnostics.py

```diff
@@ -0,0 +1,100 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for the flocker-diagnostics.
+"""
+
+import os
+from subprocess import check_output
+import tarfile
+
+from twisted.trial.unittest import TestCase
+
+from ..testtools import require_cluster
+
+# XXX: Copied from ``flocker.admin.acceptance``.
+# Should probably refactor and re-use ``flocker.admin.runner`` too.
```

Comment:
:question: Changing `admin.acceptance` to use this list would be a very small change. I encourage you to do it in this patch, because otherwise it's unlikely to happen.